### ISSUE TITLE
fix readme example

### ISSUE THREAD

**Issue by yaroslavvb** [cybertronai/autograd-lib#3]
A should be A[0].detach()
